src/sql/src/session/hint.rs

```diff
@@ -54,6 +58,8 @@ impl ApplicationNameHint {
             "dbt" => ApplicationNameHint::Dbt(Private),
             "web_console" => ApplicationNameHint::WebConsole(Private),
             "mz_psql" => ApplicationNameHint::MzPsql(Private),
+            // Note: Make sure this is kept in sync with the `fivetran-destination` crate.
+            "mz_fivetran_destination" => ApplicationNameHint::MzFivetranDestination(Private),
```

Comment:
I'd go with the more verbose `materialize_fivetran_destination` to be explicit; the `mz` in `mz_psql` is to indicate the `mz` CLI and not "Materialize" broadly.

```suggestion
            "materialize_fivetran_destination" => ApplicationNameHint::MzFivetranDestination(Private),
```

Reply:
Makes sense, updated!